app/validation/validator.py

```diff
@@ -2064,24 +2160,55 @@ def _get_answers_with_parent_ids(self, json_to_validate):
 
         return answers
 
-    def _get_questions_with_context(self, questionnaire_json):
+    @classmethod
+    def _get_option_value_to_answer_id_map(cls, json_to_validate):
+        option_value_to_answer_id_map = {}
+        questions = (
+            question
+            for question, _ in cls._get_questions_with_context(json_to_validate)
+        )
+        radio_and_checkbox_answers = cls._get_answers_by_type_from_questions(
+            questions=questions, answer_types={"Radio", "Checkbox"}
+        )
+
+        for answer in radio_and_checkbox_answers:
```

Comment:
Could a lot of this be replaced with a call to `_parse_values(json_to_validate, 'options')` to return all `options` elements?

Reply:
As discussed, `_parse_values` return a pointer and the object. Therefore, I have made a new method called `get_answers`.